<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RF_Dialog</class>
 <widget class="QDialog" name="RF_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RF_Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RF Control</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_ports">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comm. Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Comm. Ports</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_store">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Storage Location:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Gain</string>
   </property>
   <widget class="QSpinBox" name="spin_g3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 3 Gain:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_g2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 1 Gain:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_g1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 2 Gain:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Frequency Step</string>
   </property>
   <widget class="QSpinBox" name="spin_f4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1 MHz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100 MHz:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_f3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_f2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 MHz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10 MHz:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_f1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
